src/platforms/dart/configuration/integrations/logging.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Logging Integration
+caseStyle: canonical
+supportLevel: production
+sidebar_order: 3
+---
+
+The `sentry_logging` library provides [Logging](https://pub.dev/packages/logging) support for Sentry via [onRecord property](https://pub.dev/documentation/logging/latest/logging/Logger/onRecord.html) that sends events and breadcrumbs to Sentry. Once this integration is configured you can use Logging’s public API.
+
+The source can be found [on GitHub](https://github.com/getsentry/sentry-dart/tree/main/logging/lib/src).
+
+## Install
+
+To add the Logging integration, add the `sentry_logging` dependency.
+
+```yml {filename:pubspec.yaml}
+dependencies:
+  sentry: ^6.3.0
+  sentry_logging: ^6.3.0
+  logging: ^1.0.2
```

Comment:
is the package on release registry?

```suggestion
  sentry: ^{{ packages.version('sentry.dart') }}
  sentry_logging: ^{{ packages.version('sentry.logging') }}
  logging: ^1.0.2
```

Reply:
Not yet, that's why it's not using `packages.version` otherwise versions won't be aligned, I will fix that once I add it to the release registry.

Reply:
https://github.com/getsentry/sentry-release-registry/pull/66